frontend/src/components/SideNavigationTeamOptions.vue

```diff
@@ -164,17 +171,6 @@ export default {
                 }
             }
         },
-        checkFeatures () {
-            if (this.features.billing) {
-                // insert billing in second slot of admin
-                this.navigation.admin.splice(1, 0, {
```

Comment:
I belive Vue doesn't always handle deep observations, i.e. this.navigation changing will trigger a re-render, but this.navigation.admin mutating will not